### DISCUSSION TITLE
希望可以加入b服

### DISCUSSION THREAD

**Discussion by woTMrinidaye** [moesnow/March7thAssistant#321] [Q&A] [answered]
b服相交于官服只有一点点差别，就是开头登录时的火车需要一个点击登录的操作，麻烦加上吧！谢谢！

**Comment by moesnow** [comment, marked as answer]
方便提供截图（未压缩）吗，这边没有B服的测试环境。

**Reply by woTMrinidaye** [reply]
你好，非常感您的回复，这是登录界面的截图，邮件似乎会丢失图片，这是GitHub上的回复
![微信图片_20240906034641](https://github.com/user-attachments/assets/cc5219de-dfcf-4451-8eeb-3ad3c0350184)

**Reply by moesnow** [reply]
64a57580a2c7fe19ac948fe944a776ce4ae69329
功能已经加上了，下个版本可以试试

**Comment by woTMrinidaye** [comment]
新功能我使用了，点击进入之后，鼠标指针会在主页面上，导致esc操作无法执行

https://github.com/user-attachments/assets/c1e19074-0fb9-4542-8793-0d796368f8c5


![微信图片_20240906141113](https://github.com/user-attachments/assets/8dc6c14a-f46d-4d98-a5b9-8cc61f1ce913)

**Reply by moesnow** [reply]
不是很好解决欸，因为我这边没有B服环境。
是100%稳定复现吗，手动点击登录是否会这样。
鼠标指针能通过按什么键实现隐藏吗，比如Alt。

**Reply by woTMrinidaye** [reply]
额外进行一次点击就好了

**Reply by woTMrinidaye** [reply]
是100%复现的，点击进入游戏后鼠标指针就会一直在上面，测试3次在点击进入游戏后3秒再次点击屏幕都能继续执行esc任务，测试3次2秒在点击进入游戏后再次点击屏幕失败两次成功一次，应该是根据设备不同加载速度不一样，建议在点击进入游戏后5-10秒执行一次点击屏幕再进行esc的后续操作，谢谢

**Reply by woTMrinidaye** [reply]
这是b服的安装包，如果有需要可以取
https://pkg.biligame.com/games/StarRail_setup_bilibili/773823/StarRail_setup_bilibili.exe

**Reply by woTMrinidaye** [reply]
在光标悬浮在游戏表面时使用alt+tab选取崩铁也可以将光标拉进游戏，March7th Assistant程序会进行一次窗口切换到前台，所以选择相隔1分钟使用两次计划任务运行March7th Assistant就可以解决这个问题，我真聪明

**Reply by moesnow** [reply]
abbe0c92c3b6782c1e2f5a60f3bd3f5fb8380bf0
自动点击鼠标加上了，等下个版本发布可以再试试

**Reply by woTMrinidaye** [reply]
非常感谢！帮大忙了！
